HTML/mockup-ellipsis/style.css

```diff
@@ -0,0 +1,446 @@
+    color: #3c3c3c;
+    text-transform: uppercase;
+}
+
+@media only screen and (max-width: 1100px) {
+    .section__header {
+        width: 80%;
+    }
+}
+
+@media only screen and (max-width: 800px) {
+    .section__header {
+        font-size: 2.7rem;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    .section__header {
+        font-size: 2.4rem;
+    }
+}
+
+.section__header::before {
+    content: "";
+    width: 50%;
+    height: 2px;
+    position: absolute;
+    top: 50%;
+    right: 100%;
+    display: inline-block;
+    background-color: #3c3c3c;
+}
+
+@media only screen and (max-width: 1100px) {
+    .section__header::before {
+        width: 30%;
+        right: 83%;
+    }
+}
+
+.section__header.u-color-white::before {
+    background-color: #ffffff;
+}
+
+.section__header::after {
+    content: "";
+    width: 50%;
+    height: 2px;
+    position: absolute;
+    top: 50%;
+    left: 100%;
+    display: inline-block;
+    background-color: #3c3c3c;
+}
+
+@media only screen and (max-width: 1100px) {
+    .section__header::after {
+        width: 30%;
+        left: 83%;
+    }
+}
+
+@media only screen and (max-width: 800px) {
+    .section__header::after {
+        width: 30%;
+        left: 83%;
+    }
+}
+
+.section__header.u-color-white::after {
+    background-color: #ffffff;
+}
+
+.section__paragraph {
+    font-size: 1.6rem;
+    font-weight: 300;
+    line-height: 1.5;
+    text-align: center;
+    color: #3c3c3c;
+}
+
+.section__paragraph--offer {
+    margin-top: 3.7rem;
+    font-size: 1.4rem;
+    line-height: 1.85;
+}
+
+@media only screen and (max-width: 1100px) {
+    .section__paragraph--offer {
+        margin-top: 2rem;
+    }
+}
+
+.section__subheader {
+    margin-top: 4rem;
+    font-size: 2rem;
+    font-weight: bold;
+    font-style: normal;
+    font-stretch: normal;
+    line-height: normal;
+    letter-spacing: 3.2px;
+    text-align: center;
+    color: #3c3c3c;
+    text-transform: uppercase;
+}
+
+@media only screen and (max-width: 1100px) {
+    .section__subheader {
+        margin-top: 2rem;
+    }
+}
+
+.section__offer {
+    text-align: center;
+}
+
+.section__offer:not(:nth-child(2)) {
+    margin-top: 11.3rem;
+}
+
+@media only screen and (max-width: 1100px) {
+    .section__offer:not(:nth-child(2)) {
+        margin-top: 10%;
+    }
+}
+
+.section__logo-offer {
+    width: 8rem;
+}
+
+@media only screen and (max-width: 800px) {
+    .section__logo-offer {
+        width: 5rem;
+    }
+}
+
+.section-cards__list {
+    display: grid;
+    grid-template: 1fr 1fr/ 1fr 1fr 1fr;
+    row-gap: 1rem;
+    column-gap: 1rem;
+    list-style: none;
+}
+
+@media only screen and (max-width: 800px) {
+    .section-cards__list {
+        display: grid;
+        grid-template: 1fr 1fr 1fr/ 1fr 1fr;
+    }
+}
+
+@media only screen and (max-width: 500px) {
+    .section-cards__list {
+        display: grid;
+        grid-template: 1fr 1fr 1fr 1fr/ 1fr;
+    }
+}
+
+.card {
+    padding: 26% 0 5% 5%;
```

Comment:
I would place item to bottom using flexes to be sure that it will always at bottom